mozilla/fx-private-relay
# Content refresh

# New feature description

~~Do not merge this until after April 19th, so that localisers have time to deal with the unfortunate massive workload we're generating here :(~~

This applies three primary changes across the entire website:
- An "alias" is now called a "mask"
- A "real email" is now called a "true email"
- We're using "subdomain" consistently now, rather than "domain"

For that, it uses the strings from https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/66.

Fixes #1814.

# How to test

Check out https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/66, then verify that there's no mention of "alias", "real email" and "domain" anywhere anymore.

# Checklist

- [x] l10n dependencies have been merged, if any. Not yet: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/66
